pkg/operatormanager/operatormanager.go

```diff
@@ -570,10 +577,13 @@ func (m *OperatorManager) createOrUpdateExporterSidecarService(ctx context.Conte
 	}
 	pes.Spec.Ports = []v1.ServicePort{
 		{
-			Name:       "metrics",
-			Port:       int32(exporterServicePort),
-			Protocol:   v1.ProtocolTCP,
-			TargetPort: pg.ExporterSidecarPortName,
+			Name:     "metrics",
+			Port:     int32(exporterServicePort),
+			Protocol: v1.ProtocolTCP,
+			TargetPort: intstr.IntOrString{
+				Type:   intstr.String,
+				StrVal: "exporter",
```

Comment:
Hmm, maybe derive this value from the container definition? If someone changes the configmap an renames this port there, this service won't work anymore...

Reply:
The configurable `.Values.sidecars.exporter.containerPort` in *values.yaml* is used now.